src/documents/migrations/1055_alter_storagepath_path.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 5.1.1 on 2024-10-03 14:47
+
+from django.db import migrations
+from django.db import models
+from django.db import transaction
+
+
+def convert_from_format_to_template(apps, schema_editor):
+    # TODO: Is there a signal to disable?  I don't want documents getting moved while this is running
```

Comment:
@shamoon Any ideas about this?  I swear it exists, but I cannot seem to recall where

Reply:
Hmm, you mean https://github.com/paperless-ngx/paperless-ngx/blob/3d02f54712bc87f7de46480e01650f8f486c32a5/src/documents/signals/handlers.py#L367 ? I think that's only for docs, not sure if theres another you're thinking of. I think that's what renamer does, triggers the post_save signal: https://github.com/paperless-ngx/paperless-ngx/blob/3d02f54712bc87f7de46480e01650f8f486c32a5/src/documents/management/commands/document_renamer.py#L25